Desktop_Interface/isobuffer.cpp

```diff
@@ -2,93 +2,92 @@
 #include "isodriver.h"
 #include "uartstyledecoder.h"
 
-isoBuffer::isoBuffer(QWidget* parent, int bufferLen, isoDriver* caller, unsigned char channel_value)
-	: QWidget(parent)
-{
-	buffer = (short*) calloc(bufferLen*2, sizeof(short));
-	bufferEnd = bufferLen-1;
-	samplesPerSecond = (double) bufferLen/(double)21;
-	samplesPerSecond = samplesPerSecond/375*VALID_DATA_PER_375;
-	sampleRate_bit = samplesPerSecond * 8;
-	virtualParent = caller;
-	channel = channel_value;
+namespace {
+	static char const * fileHeaderFormat =
+		"EspoTek Labrador DAQ V1.0 Output File\n"
```

Comment:
I've never thought about spreading text across lines like that, and I've never seen it either.  But now that I have, it makes so much sense to do it that way.